sega_genesis_checksum_utility_for_python_3.py

```diff
@@ -25,46 +25,46 @@ def main():
                                             'Genesis ROM binary.')
   args = parser.parse_args()
 
-  print 'Reading checksum from file...'
-  with open(args.path[0], 'r+b') as genesis_file:
+  print('Reading checksum from file...')
+  with open(args.path[0], 'rb') as genesis_file:
     valid_genesis_file = verify_console_name_from_header(genesis_file)
     if not valid_genesis_file:
-      print '\nERROR: File is not a valid Genesis or Mega Drive ROM file.'
+      print('\nERROR: File is not a valid Genesis or Mega Drive ROM file.')
       return
 
     header_checksum = read_checksum_from_header(genesis_file)
-    print 'Header checksum =',
+    print('Header checksum =',)
     print_word(header_checksum)
 
-    print 'Computing checksum...'
+    print('Computing checksum...')
     computed_checksum = compute_checksum(genesis_file)
-    print 'Computed checksum = ',
+    print('Computed checksum = ',)
     print_word(computed_checksum)
 
     if header_checksum == computed_checksum:
-      print '\nChecksums match. Horray!'
+      print('\nChecksums match. Horray!')
       return
 
-    print '\nWARNING: Checksums do not match!'
+    print('\nWARNING: Checksums do not match!')
     while(1):
-      data = raw_input('\nWould you like to update the header checksum to match'
+      data = input('\nWould you like to update the header checksum to match'
                        'match the computed checksum? (y/n)')
```

Comment:
nitpick: Fix alignment of the strings for `input()`

Reply:
Didn't understood properly about what you mean. But in python3 the raw_input() function is not available. It is merged into input()

Reply:
Sorry for not being more clear.

This is not a problem with the code itself, but just how it's formatted.

Can we remove the spaces on the wrapped line so that the quotes of the text are aligned?

e.g. https://i.imgur.com/xjqEBTn.jpg